cloud/blockstore/libs/storage/partition_nonrepl/part_nonrepl_common.cpp

```diff
@@ -34,12 +34,13 @@ void ProcessError(
 {
     if (error.GetCode() == E_BS_INVALID_SESSION) {
         SendEvReacquireDisk(system, config.GetParentActorId());
-
         error.SetCode(E_REJECTED);
     }
 
     if (error.GetCode() == E_IO || error.GetCode() == MAKE_SYSTEM_ERROR(EIO)) {
         error = config.MakeIOError(std::move(*error.MutableMessage()));
+    } else {
+        config.AugmentErrorFlags(error);
```

Comment:
one of the augmentations in AugmentErrorFlags is
```c++
        if (MuteIOErrors) {
            SetErrorProtoFlag(error, NCloud::NProto::EF_HW_PROBLEMS_DETECTED);
        }
```
but it will be completely skipped if previous condition is taken

Reply:
in this branch 
MakeIOError()->MakeError()->AugmentErrorFlags()